src/core/expr/fexpr.cc

```diff
@@ -669,6 +671,69 @@ DECLARE_METHOD(&PyFExpr::first)
     ->docs(doc_first);
 
 
+static const char* doc_as_type =
+R"(as_type()
+--
+
+Equivalent to :func:`dt.as_type(self)`.
+)";
```

Comment:
I mean, instead of declaring `doc_as_type` variable here, you can declare an `extern const char* doc_FExpr_as_type` in file "documentation.h", and the actual content of the documentation put into the file `docs/api/fexpr/as_type.rst`.

Reply:
Currently getting an error message when previewing the docs : 
```(test_env) [sam@fedora docs]$ make html
Running Sphinx v4.0.2
loading intersphinx inventory from https://docs.python.org/3/objects.inv...
loading intersphinx inventory from https://ipython.readthedocs.io/en/stable/objects.inv...
building [mo]: targets for 0 po files that are out of date
building [html]: targets for 408 source files that are out of date
updating environment: [new config] 408 added, 0 changed, 0 removed
/home/sam/Documents/datatable/docs/api/type/list32.rst:7: WARNING: Could not find method `PyType::list32` in file src/core/types/py_type.cc                                       
/home/sam/Documents/datatable/docs/api/type/list64.rst:7: WARNING: Could not find method `PyType::list64` in file src/core/types/py_type.cc

Exception occurred:
  File "/home/sam/Documents/datatable/docs/_ext/xpython.py", line 251, in get_title_and_target
    assert ctx[1].startswith(xpy.main_module)
AssertionError
The full traceback has been saved in /tmp/sphinx-err-fbyr6ti9.log, if you want to report the issue to the developers.
Please also report this if it was a user error, so that a better error message can be provided next time.
A bug report can be filed in the tracker at <https://github.com/sphinx-doc/sphinx/issues>. Thanks!
make: *** [Makefile:25: html] Error 2
```

I have also attached the error log
[sphinx_error.log](https://github.com/h2oai/datatable/files/6839538/sphinx_error.log)

Reply:
please merge with latest main and see if the errors still there

Reply:
Hi @st-pasha ; merged with the latest main, but the error still persists: 
```(dev) [sam@fedora datatable]$ cd docs
(dev) [sam@fedora docs]$ make html
Running Sphinx v4.1.1
loading intersphinx inventory from https://docs.python.org/3/objects.inv...
loading intersphinx inventory from https://ipython.readthedocs.io/en/stable/objects.inv...
building [mo]: targets for 0 po files that are out of date
building [html]: targets for 409 source files that are out of date
updating environment: [new config] 409 added, 0 changed, 0 removed
reading sources... [ 99%] releases/v1.1.0                                                                                                                                         
Exception occurred:
  File "/home/sam/Documents/datatable/docs/_ext/xpython.py", line 250, in get_title_and_target
    assert ctx[1].startswith(xpy.main_module)
AssertionError
The full traceback has been saved in /tmp/sphinx-err-n5pxp83i.log, if you want to report the issue to the developers.
Please also report this if it was a user error, so that a better error message can be provided next time.
A bug report can be filed in the tracker at <https://github.com/sphinx-doc/sphinx/issues>. Thanks!
make: *** [Makefile:25: html] Error 2
(dev) [sam@fedora docs]$ 
```

Reply:
i pushed a fix